stubbs module tests and execution fails when parent shell is non-bash

line 1030 of rerun:

```
    local -r RERUN_SHELL=$SHELL
```

if a user is using a non-bash shell (e.g. zsh) and then runs a bash process, which may not override the $SHELL variable but instead still point to the other shell, then rerun will invoke that shell resulting in test and execution failures:

env
```
$ echo $SHELL
/usr/local/bin/zsh
```

command
```
$ rerun stubbs:add-module -m "example-module" --description "example module which fails"    
```
output
```                           
/Users/username/Documents/dev/rerun/rerun:117: no matches found: [0]=debug
/Users/username/Documents/dev/rerun/modules/stubbs/lib/functions.sh:14: command not found: rerun_log
```

command
```
$ make check
```
output
```
<snip>
echo "Testing stubbs module..."; \
	if [[ "${PWD}x" = "x" ]]; then export PWD="/Users/username/Documents/dev/rerun"; fi; \
	if [[ "." = "/*" ]]; then MYDIR="."; else MYDIR="${PWD}/."; fi; \
	export RERUN="${MYDIR}/rerun"; \
	export RERUN_MODULES="${MYDIR}/modules"; \
	export LC_ALL="C"; \
	echo "    modules at ${RERUN_MODULES}"; \
	if [ ! -w ${RERUN_MODULES} ]; then \
		CLEANUPDIRS="true"; \
		chmod u+w ${RERUN_MODULES} ${RERUN_MODULES}/stubbs/tests; \
	fi; \
	set -e; \
	${RERUN} stubbs:test --module stubbs; \
	if [[ "${CLEANUPDIRS}" = "true" ]]; then \
		chmod u-w ${RERUN_MODULES} ${RERUN_MODULES}/stubbs/tests; \
	fi
Testing stubbs module...
    modules at /Users/username/Documents/dev/rerun/./modules
/Users/username/Documents/dev/rerun/rerun:117: no matches found: [0]=debug
/Users/username/Documents/dev/rerun/./modules/stubbs/lib/functions.sh:14: command not found: rerun_log
make: *** [test-stubbs] Error 1
```

Will open a PR with a fix

--- a/rerun
+++ b/rerun
@@ -1027,7 +1027,7 @@ rerun_command_execute() {
     rerun_script_exists "${RERUN_MODULE_DIR:-}" $command || {
         rerun_syntax_error "command not found: \"$module:$command\""
     }
-    local -r RERUN_SHELL=$SHELL
+    local -r RERUN_SHELL=$(which bash)
     local COMMAND_SHELL=$(rerun_property_get "$RERUN_MODULE_DIR" SHELL)
     : ${COMMAND_SHELL:=$RERUN_SHELL}; # default it.
 

